app/src/main/java/edu/akelael/comics/model/ComicListModel.java

```diff
@@ -0,0 +1,48 @@
+package edu.akelael.comics.model;
+
+import java.util.HashMap;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class ComicListModel implements IComicListModel {
+    Server server;
+    Retrofit retrofit;
+    HashMap<String, String> mMap;
+
+    @Override
+    public void initializeApiConnection(){
+        Retrofit.Builder retrofitBuilder;
+        retrofitBuilder = new Retrofit.Builder();
+        retrofitBuilder.baseUrl(server.baseUrl);
+        retrofitBuilder.addConverterFactory(GsonConverterFactory.create());
+        retrofit = retrofitBuilder.build();
+        mMap = new HashMap<>();
+        mMap.put("ts", "ts");
+        mMap.put("apikey", "apikey");
+        mMap.put("hash", "hash");
+    }
+
+    @Override
+    public void getComics(final IComicListModelCallbacks listener){
+        server = retrofit.create(Server.class);
```

Comment:
This call can be moved to the initializeApiConnection method, so it will be called only once. It makes sense IMO, as now a Server object is created each time you retrieve the comics list, and this movement can considered as a part of the api initialization process. Also, if what I propose is done, there's no need for the "retrofit" field to be a field anymore (it can be made a local variable).